.idea/misc.xml

```diff
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_14" project-jdk-name="14" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="false" project-jdk-name="1.8.0_241" project-jdk-type="JavaSDK">
```

Comment:
You don't have to commit all. This can be simply not selected in the changelist.